ocp_resources/resource.py

```diff
@@ -1086,3 +1092,14 @@ def _apply_patches(patches, action_text, action):
                 resource.update_replace(
                     resource_dict=patch
                 )  # replace the resource metadata
+
+    def _apply_patches_sampler(self, patches, action_text, action):
+        exceptions_dict = {ConflictError: []}
+        exceptions_dict.update(DEFAULT_CLUSTER_RETRY_EXCEPTIONS)
+        return Resource.retry_cluster_exceptions(
+            func=self._apply_patches,
+            exceptions_dict=exceptions_dict,
+            patches=patches,
```

Comment:
The `patches` variable will be modified by _apply_patches, please add a .copy() unless this is intentional.

Reply:
@sarahbx This update seems intentional. I also want to add, my PR is not changing anything in  `_apply_patches `, so If this comment needs to be addressed, it should be addressed in a separate PR.

With TimeoutSampler changes taken out of IUO area, without this piece to handle conflict error, we can have significant fail rate in current release testing.